### ISSUE TITLE
Issue: Tile blends into background theme

### ISSUE THREAD

**Issue by tecosaur** [Kruemelkatze/vscode-dashboard#8]
Here's a demo

![image](https://user-images.githubusercontent.com/20903656/47950399-b29f4d00-df8c-11e8-95af-23a61e3e1f74.png)

Could definitely look better. Perhaps using something like a transparent grey may work well? (e.g. `rgba(128, 128, 128, 0.2)`

**Comment by Kruemelkatze** [comment]
Hey!
Sorry for not responing earlier! Are you using a custom theme? The dashboard uses only colors defined by the user's vcode theme. I tested against all default themes and it look nicely on all of them.

However, as many custom themes may produce such an problem, I included settings for customizing the 3 main colors in the dashboard. :)

Example usage:
![dashboard-settings](https://user-images.githubusercontent.com/5564731/49552538-5a919880-f8f4-11e8-8b39-4bf4370df38c.PNG)
![dashboard-colors](https://user-images.githubusercontent.com/5564731/49552541-5cf3f280-f8f4-11e8-9fef-633fc11eb208.PNG)

As this (included in v1.2) should fix your problem, I'm closing this issue.
